src/view3d.cpp

```diff
@@ -228,8 +228,11 @@ std::unique_ptr<QGLFramebufferObject> View3D::allocIncrementalFramebuffer(int w,
     // for samples==1, so work around it by forcing 0, if possible
     fboFmt.setSamples(fmt.samples() > 1 ? fmt.samples() : 0);
     //fboFmt.setTextureTarget();
-    return std::unique_ptr<QGLFramebufferObject>(
-        new QGLFramebufferObject(w, h, fboFmt));
+    std::unique_ptr<QGLFramebufferObject> fbo;
+    fbo.reset(new QGLFramebufferObject(w, h, fboFmt));
+    if (fbo.get() && fbo->attachment()==QGLFramebufferObject::NoAttachment)
```

Comment:
I merged before I noticed this anyway, and don't want to bother you for another trivial change in any case.